### PR TITLE
Added target=_blank attribute to appropriate links.

### PR DESCRIPTION
**Pull request by Mateusz1223**
Fixes #

#### Changes proposed in this pull request:

-

- Link to live demo: http://pr-XXX-evalai.surge.sh 

-

### CODE REVIEW

File: src/app/components/get-involved/get-involved.component.html [hunk 1]
```diff
@@ -93,4 +93,4 @@ <h4 class="get-involved-title align-center fw-regular">Get Involved</h4>
     &nbsp;
   </div>
 </div>
-<app-footer></app-footer>
+<app-footer></app-footer>
```
Comment (Sanji515) [inline src/app/components/get-involved/get-involved.component.html]: @Mateusz1223 Can you please add a new line here?

**Review by Sanji515 — COMMENTED** [review]
Just a minor comment. Rest LGTM!

**Comment by Sanji515** [comment]
Also, @Mateusz1223 Please update the PR description.

File: src/app/components/about/about.component.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 			<div class="col-lg-7 col-md-7 col-sm-12 col-xs-12 about-content-column">
 				<h4 class="about-title">About Us</h4>
 				<p class="about-content">EvalAI is built by a team of open source enthusiasts working at 
-					<a href="http://cloudcv.org" class="blue-text">CloudCV</a>. CloudCV aims to make AI research 
+					<a href="http://cloudcv.org" target="_new" class="blue-text">CloudCV</a>. CloudCV aims to make AI research
```
Comment (Ram81) [inline src/app/components/about/about.component.html]: @Mateusz1223 please add `target="_blank"` instead of `target="new"`

**Review by Ram81 — CHANGES_REQUESTED** [review]

**Review by Ram81 — APPROVED** [review]
LGTM 👍

**Review by Sanji515 — APPROVED** [review]

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**